cmd/mvpage/main.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 Istio Authors
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	rootCmd := cobra.Command{
+		Use:   "mvpage <old markdown file> <new markdown file>",
+		Short: "Moves a content page within a Hugo site.",
+		Long: "Moves a content page within a Hugo site, updating all links to the page\n" +
```

Comment:
Can you add the usage content to a readme.md in this directory?

Reply:
Done.